### ISSUE TITLE
Hired kerbals swapping between sessions

### ISSUE THREAD

**Issue by JEGHC** [godarklight/DarkMultiPlayer#421]
### Environment
- OS: Windows 10
- KSP: v1.2.2
- DMP: v0.2.4.2

### Description
List of hired kerbals keeps changing between sessions. Resulting in lost experience in career mode. The overall number of hired kerbals seems to stay roughly constant but it just seems to swap out kerbals for new ones. It also doesn't do this to all the kerbals. Just a random handful between each play session. 

### Additional Info
I am wondering if it is a naming issue where other people on the server are assigned kerbals with the same name? If it was this, how easy would it be to have the kerbals spawned with a surname that matched the players user-name? Re-naming kerbals post-hire doesn't seem to help alleviate the issue at all. The server has mods enabled so if no-one else is seeing this issue then I would assume it is some other mod allowing the Kraken to enter.

**Comment by 81ninja** [comment]
You're right, it's a known issue caused by players sharing the same kerbals. And the way that works right now is far from perfect.
Assigning kerbals different names won't work because when people launch crafts, the kerbals they contain must exist in everybody's game. So you'd end up just having a lot more kerbals in the server.
You can try disabling reverts, as that is known to cause kerbals to go MIA and die upon reloading.

There are proposed fixes to this, but none into which work has yet started.
